suite-native/atoms/src/Pictogram/CriticalShapeSvg.tsx

```diff
@@ -0,0 +1,22 @@
+import Svg, { Path } from 'react-native-svg';
+
+import { useNativeStyles } from '@trezor/styles';
+
+export const CriticalShapeSvg = () => {
+    const { utils } = useNativeStyles();
+    const borderColor = utils.colors.backgroundAlertRedSubtleOnElevationNegative;
```

Comment:
why has  the `CriticalIconSvg` customizable color prop but the `...Shape...` does not? Would not make sense to unify it? I would expect that the shape can change to color as well.

Reply:
Because the shapes have the color predefined, we agreed with @shenkys that we won't allow changing it. The `*IconSvg` have configurable `color` so the logic for them and `Icon` is in one place. I could extract colors of the shapes into the config object but I'm not sure there's any meaningful benefit. Would you prefer that more?

Reply:
Initially I had parametrized `CircleShapeSvg` which had more lines of code than `SuccessShapeSvg` and `InfoShapeSvg` combined. And was also struggling with naming the rest (e.g. `TriangleShapeSvg` and `HexagonShapeSvg`) so I decided to make it easier.